### ISSUE TITLE
Buttons on comments are a really small hit targets

### ISSUE THREAD

**Issue by fsih** [LLK/scratch-gui#2288]
### Expected Behavior
Buttons should have a minimum size for accessibility

### Actual Behavior
It's kind of hard to hit the minimize button on comments. Also resize and close.
![minimizecomment](https://user-images.githubusercontent.com/2855464/41167582-59986092-6b11-11e8-9f6c-dad7a15a25ee.gif)

### Steps to Reproduce
Try to minimize comments
Here is a project
https://llk.github.io/scratch-gui/smoke/#223875256

### Operating System and Browser
Windows Edge

**Comment by benjiwheeler** [comment]
+1 on resizing target being too small:

Argh!!

![comment resize](https://user-images.githubusercontent.com/3431616/50643181-482e6d00-0f3b-11e9-9955-1c263d2884a4.gif)

**Comment by paulkaplan** [comment]
This has been adjusted. If it continues to be a problem, let's re-open or create a new issue.
